examples/timeline/other/usingReact.html

```diff
@@ -0,0 +1,131 @@
+    <script src="https://fb.me/react-dom-15.0.1.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/babel-core/5.8.23/browser.min.js"></script>
+
+    <script src="../../../dist/vis.js"></script>
+    <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+    
+    <!--
+    This is where you link to your React code. Can be .js or .jsx 
+    extension, doesn't really matter.
+    -->
+    <script type="text/babel">
+        var timeline;
+        
+        // create groups
+        var numberOfGroups = 25; 
+        var groups = new vis.DataSet()
+        for (var i = 0; i < numberOfGroups; i++) {
+            groups.add({
+              id: i,
+              content: 'Truck ' + i
+            })
+        }
+          
+        // create items
+        var numberOfItems = 1000;
+        var items = new vis.DataSet();
+        var itemsPerGroup = Math.round(numberOfItems/numberOfGroups);
+        for (var truck = 0; truck < numberOfGroups; truck++) {
+            var date = new Date();
+            for (var order = 0; order < itemsPerGroup; order++) {
+              date.setHours(date.getHours() +  4 * (Math.random() < 0.2));
+              var start = new Date(date);
+              date.setHours(date.getHours() + 2 + Math.floor(Math.random()*4));
+              var end = new Date(date);
+              items.add({
+                id: order + itemsPerGroup * truck,
+                group: truck,
+                start: start,
+                end: end,
+                content: 'Order ' + order
+              });
+            }
+        }
+
+        var GroupTemplate = React.createClass({
+          render: function() {
+            var { group } = this.props;
+            return <div>
+              <label>{group.content}</label>
+              <button>do stuff</button>
+            </div>
+          }
+        })
+
+        var ItemTemplate = React.createClass({
+          render: function() {
+            var { item } = this.props;
+            return <div>
+              <label>{item.content}</label>
+              <button>do stuff</button>
+            </div>
+          }
+        })
+
+        // specify options
+        var options = {
+          rtl: true,
+          stack: true,
+          maxHeight: 400,
+          start: new Date(),
+          end: new Date(1000*60*60*24 + (new Date()).valueOf()),
+          editable: true,
+          margin: {
+            item: 10, // minimal margin between items
+            axis: 5   // minimal margin between items and the axis
+          },
+          template: function (item, element) {
+            ReactDOM.unmountComponentAtNode(element);
+            return ReactDOM.render(<ItemTemplate item={item} />, element);
+          },
+          groupTemplate: function (group, element) {
+            ReactDOM.unmountComponentAtNode(element);
+            return ReactDOM.render(<GroupTemplate group={group} />, element);
+          },
+          orientation: 'top'
```

Comment:
I needed a second to realize that the "orientation" option had nothing to do with the template function. I would put it on top (e.g. line 93) to separate it from the "groupTemplate" option.